### ISSUE TITLE
TypeError: require(...) is not a function

### ISSUE THREAD

**Issue by s-panferov** [postcss/postcss-selector-matches#9]
Hi, I have an issue with `2.0.2` version:

```
node_modules/postcss-cssnext/lib/features.js:101
    return require("postcss-selector-matches")(options);
                                              ^
TypeError: require(...) is not a function
```

**Comment by MoOx** [comment]
Closed by d6ce44b7091884f831f5b56ddc42c2b30aa35c3d

**Comment by s-panferov** [comment]
Thanks for the quick fix!
